src/ConnectionInterface.php

```diff
@@ -1,11 +1,18 @@
 <?php
 
+// vim:ts=4:sw=4:et:fdm=marker:fdl=0
+
 namespace atk4\data;
 
+/**
+ * Class description?
```

Comment:
No, currently we don't use this interface anywhere. Can I delete it completely in this PR?